website/docs/reference/segments.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Segments
+---
+
+:::info Availability
+Strategy constraints are available to Unleash Pro and Enterprise users from Unleash 4.10 and onwards.
+:::
+
+A **segment** is a reusable collection of [strategy constraints](../advanced/strategy-constraints.md).
+Like with strategy constraints, you apply segments to [feature toggle activation strategies](../user_guide/activation-strategies.md).
+
+You can apply the same segment to multiple activation strategies.
+If you update the segment, the changes will affect every strategy that uses that segment.
+
+
+## Structure and evaluation
+
+Segments are collections of strategy constraints. To satisfy a segment, *all* the constraints in the collection must be satisfied.
+
+If an activation strategy has a segment *and* additional constraints applied, the segment *and* the strategies must all be satisfied.
+
+## Creating, updating, and deleting segments
+
+Segments can be created, edited, and deleted from the segments page in the admin UI or via the API (see the [segments API documentation](../api/admin/segments.mdx)).
+
+A segment that is in use **cannot** be deleted. If you'd like to delete a segment that is in use, you must first remove the segment from all the activation strategies that are currently using it.
+
+[image of the segments page and the segments item in the menu?]
+
+## When to use segments
```

Comment:
Should we move this to the top? Nice with motivating examples before technical details.

Reply:
Maybe not that image, but we could definitely have a header image of sorts, yeah!

Reply:
Ah I meant the `When to use segments` section, or maybe just parts of it. Not sure.

Reply:
Oh, right! My bad 😅 Not a bad idea. I'll look into it! 🙋🏼

Reply:
Very nice :+1:

Reply:
Done! Thanks 😄